README.md

```diff
@@ -0,0 +1,29 @@
+# SQLite Speedtest with Emscripten
+
+This repository shows how to compile and run the speedtest1 from the SQLite
+project in the browser using different storage backends.
```

Comment:
Consider mentioning the backends and explaining a bit of the results e.g.

"MEMFS shows faster execution since database files are only kept in memory. Similarly, IDBFS shows faster execution, since operations are not actually persisted from within the Wasm module (see [FS.syncfs](https://emscripten.org/docs/api_reference/Filesystem-API.html#FS.syncfs))"

Reply:
Done.